packages/ifpack2/src/Ifpack2_Parameters.cpp

```diff
@@ -64,6 +67,17 @@ void getValidParameters(Teuchos::ParameterList& params)
   params.set("fact: iluk level-of-fill", (int)1);
   params.set("fact: iluk level-of-overlap", (int)0);
 
+  // Ifpack2_Chebyshev
+  params.set("chebyshev: max eigenvalue",STS::nan());
+  params.set("chebyshev: ratio eigenvalue",STS::nan());
+  params.set("chebyshev: min eigenvalue",(double)30.0);
```

Comment:
1. Avoid C-style casts; use C++ casts (either `double(30.0)` or `static_cast<double>(30.0)` -- I prefer the latter for integer casts, which may be a source of bugs).
2. The type of the literal `30.0` is `double`.

Reply:
Have a look at the rest of the file. Just trying to be consistent here. (But we should change a lot of that once we rework that, I agree.)